storage-inventory/ops/README.md

```diff
@@ -2,118 +2,169 @@
+- The terms _collection_ and _namespace_ are occasionally used interchangeably, but _namespace_ has a key role in artifact metadata whereas _collection_ is just a term used to understand how artifacts are logically grouped.
+
+<a id='term-namespace'>**namespace**</a>  
+A _namespace_ is an SI identifier for a _collection_ or part of a _collection_.  For example, `cadc:CFHT/` might be used to identify all CFHT files held at the CADC (the CFHT 'collection'); `cadc:CFHT/raw/` might be used to identify all of the raw CFHT files held at the CADC -- both `cadc:CFHT/` and `cadc:CFHT/raw/` are _namespaces_ but they identify different scopes of the CFHT _collection_.  SI services and applications often act on namespaces which are defined using regex patterns -- such as during file replication and determining access permissions -- so some thought must be put into what is used.  See the [SI data model page](https://github.com/opencadc/storage-inventory/tree/master/storage-inventory-dm) for more detail on the concepts of _URI_ and _namespace_.
+- when matching a namespace to an artifact URI, URIs are parsed using `:` and `/` as delimiters.
+    - ❗A namespace must always end at a delimiter to avoid confusion with other namespaces (e.g. cadc:TEST and cadc:TESTDATA, instead of cadc:TEST/ and cadc:TESTDATA/).
+- the URI for an artifact is '_namespace/filepath_'.  A _namespace_ which includes an artifact can include any delimited path defined by the artifact's URI, e.g. all of the following are namespaces which include the artifact `cadc:TEST/data/raw/example/test1.fits`:
+    - `cadc:`
+    - `cadc:TEST/`
+    - `cadc:TEST/data/`
+    - `cadc:TEST/data/raw/`
+    - `cadc:TEST/data/raw/example/`
+- ❗ A _URI_ which has a _filepath_ of just the _filename_ (e.g. `cadc:test1.fits`) is not recommended.
+- The part of the namespace preceding the `:` is referred to as the 'scheme' elsewhere in the documentation.  This could be thought of as defining a group of collections (e.g. `cadc:DAO`, `cadc:CFHT`, `cadc:JCMT`)
+- The namespace hierarchy should be chosen to suit your project or telescope operations: a practical example of defining a namespace for a collection _might_ be:
+    - `cadc:DAO/` - the namespace for all files from the DAO telescope which are unique to the CADC.
+    - `cadc:DAO/camera1/raw/` - the namespace for all files from the DAO which are unprocessed raw observations using camera 1.
+    - `cadc:DAO/camera1/raw/biases/` - the namespace for the subset of all camera 1 raw files which are bias frames.
+    - `cadc:DAO/camera1/processed/` - the namespace for the subset of all processed frames from camera 1.
+    - `cadc:DAO/pi2023011/` - the namespace for the PI project designated '2023011'.
+    - alternatively, you could choose a simple flat namespace for all files, e.g. `cadc:DAO/`.
+
+<a id='term-resourceid'>**resourceID**</a>  
```

Comment:
I think most people looking at SI will not be familiar with IVOA registries (or IVOA, period).  The fact that these are IVOA compatible is a useful bit of info, but secondary to SI.  There was a lot of confusion about what these meant, and what had to go into them.

Reply:
I'm not sure, but I don't think it would be necessary.  `reg` itself is an ivoa compliant registry, but I don't think the services registered with it need to be.  That being said, I don't thing reg implements the entire registry spec....